draft-ietf-tls-tls13.md

```diff
@@ -2159,7 +2176,15 @@ Although X9.62 supports multiple point formats, any given curve
 MUST specify only a single point format. All curves currently
 specified in this document MUST only be used with the uncompressed
 point format (the format for all ECDH functions is considered
-uncompressed).
+uncompressed). Peers MUST validate each other's public value Y by ensuring
+that the point is a valid point on the elliptic
+curve. For all elliptic curves except for x25519 and x448, the appropriate
```

Comment:
This reads as requiring X25519 and X448 to validate Y too, albeit with a different procedure that for the NIST curves, but that procedure is not linked. While X25519/X448 are supposed to be twist-secure, does this addition require point validation for them? If so, the technique should be referenced here.